### PR TITLE
Sub: correct LOIT parameter documentation prefix

### LINKED ISSUE #11769: Sub: Loiter parameters are prefixed with LOITER_ instead of LOIT_
In SITL, `param show` outputs:
```
...
LOITER_ACC_MAX   487.640015
LOITER_ANG_MAX   0.000000
LOITER_BRK_ACCEL 250.000000
LOITER_BRK_DELAY 1.000000
LOITER_BRK_JERK  500.000000
LOITER_SPEED     1170.000000
...
```
(Both `LOITER_BRK_ACCEL` and `LOITER_BRK_DELAY` reach the 16 chars limit, but none exceeds it)

But param_parse generates this:
```
...
<parameters name="LOIT_">
<param humanName="Loiter Angle Max" name="LOIT_ANG_MAX" documentation="Loiter maximum lean angle. Set to zero for 2/3 of PSC_ANGLE_MAX or ANGLE_MAX" user="Advanced">
<field name="Units">deg</field>
<field name="UnitText">degrees</field>
<field name="Range">0 45</field>
<field name="Increment">1</field>
</param>
...
```

This seems to be the root of the issue: https://github.com/ArduPilot/ardupilot/blob/master/ArduSub/Parameters.cpp#L435

Seen in master and ardusub-beta running in SITL

### PR DESCRIPTION
**Pull request by peterbarker**
Closes #11769

### CODE REVIEW

**Comment by rmackay9** [comment]
To be consistent with Copter I think we should actually rename the LOITER_ parameters to be LOIT_.

I know that means a parameter name change but I suspect most sub users are not really using Loiter so it will be ok.

**Comment by peterbarker** [comment]
On Wed, 10 Jul 2019, Randy Mackay wrote:

> To be consistent with Copter I think we should actually rename the LOITER_ parameters to be LOIT_.

I'm not fussed one way or the other :-)

I'd just like the docco to be consistent with the code...

**Comment by jaxxzer** [comment]
I'm with @rmackay9 as these haven't been used before. I hope it will eventually help with search terms.

File: ArduSub/Parameters.cpp [hunk 1]
```diff
@@ -430,7 +430,7 @@ const AP_Param::Info Sub::var_info[] = {
     // @Path: ../libraries/AC_WPNav/AC_WPNav.cpp
     GOBJECT(wp_nav, "WPNAV_",       AC_WPNav),
 
-    // @Group: LOIT_
+    // @Group: LOITER_
     // @Path: ../libraries/AC_WPNav/AC_Loiter.cpp
     GOBJECT(loiter_nav, "LOITER_", AC_Loiter),
```
Comment (jaxxzer) [inline ArduSub/Parameters.cpp]:
```suggestion
    GOBJECT(loiter_nav, "LOIT_", AC_Loiter),
```

File: ArduSub/Parameters.cpp [hunk 2]
```diff
@@ -430,7 +430,7 @@ const AP_Param::Info Sub::var_info[] = {
     // @Path: ../libraries/AC_WPNav/AC_WPNav.cpp
     GOBJECT(wp_nav, "WPNAV_",       AC_WPNav),
 
-    // @Group: LOIT_
+    // @Group: LOITER_
```
Comment (jaxxzer) [inline ArduSub/Parameters.cpp]:
```suggestion
    // @Group: LOIT_
```

**Comment by peterbarker** [comment]
@jaxxzer squished it together

**Pull request merged**